thirdParty/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@
 
 if(BUILD_TESTING)
     if(alpaka_USE_INTERNAL_CATCH2)
-        message(STATUS "Catch2: Using INTERNAL version 3.0.1")
+        message(STATUS "Catch2: Using INTERNAL version 3.1.1")
         # Force Catch2's CMake to pick up the variables we set below
         set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
```

Comment:
Is this still required?

Reply:
Yes, until Catch2 upgrades to (at least) CMake 3.13 (in this release they upgraded from 3.5 to 3.10). Otherwise the Catch2 CMake will just ignore the variables set below this line.